.github/CODEOWNERS

```diff
@@ -5,14 +5,18 @@
 /src/platforms/android/ @getsentry/team-mobile
 /src/platforms/apple/ @getsentry/team-mobile
 /src/platforms/flutter/ @getsentry/team-mobile
+/src/platforms/dart/ @getsentry/team-mobile
 /src/platforms/react-native/ @getsentry/team-mobile
 
 /src/platforms/dotnet/ @bruno-garcia
-/src/platforms/java/ @bruno-garcia
+/src/platforms/java/ @bruno-garcia @marandaneto
 
 /src/platforms/native/ @getsentry/owners-native
 
 /src/platforms/rust/ @Swatinem
 
 /src/docs/product/discover-queries/ @getsentry/visibility
 /src/docs/product/performance/ @getsentry/visibility
+
+/src/docs/product/cli/ @getsentry/team-webplatform
```

Comment:
This is probably a bit much, how about we restrict this to the releases commands?

If you don't mind, I would add the native team in the same PR?

Reply:
@jan-auer sure, go for it.
pay attention that the sorting here matters, the last match is the one that will be assigned as reviewer.

Reply:
Ok so the problem is that I cannot select @getsentry/owners-native as a reviewer team in sentry-docs. Probably a permission issue. 

@PeloWriter to you have admin on this repo to look into this?

Reply:
@jan-auer - owners-native needs to be granted permissions to the docs repo, which should be handled by an IT request (for compliance/security reasons).

Reply:
We're probably going to address this by moving the @getsentry/codeowners team under @getsentry/engineering, which should take care of this.

Reply:
👍